sql/core/src/test/resources/sql-tests/inputs/pgSQL/int2.sql

```diff
@@ -0,0 +1,129 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- INT2
+-- https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/int2.sql
```

Comment:
Sorry, but could you check `REL_12_BETA2` and update this to `REL_12_BETA2`?

Reply:
Sure.